guides/Git Shopify.md

```diff
@@ -0,0 +1,50 @@
+# Git strategy
+
+Our quick guide to working collaboratively with Git, GitHub and Shopify projects.
+
+## Create a Feature
+
+Check if `staging` branch present in repo.
```

Comment:
Hahha, I think this is the nuance of the Russian language. Let's change this paragraph to:

Create a `staging` branch from `master`.

```
git checkout master
git pull
git checkout -b staging
```
If the staging branch already exists, rebase it on-top of master:

```
git checkout staging
git rebase master
```

Finally, push the staging branch to remote so we have a communal copy:
```
git push origin staging
```